.github/workflows/release.yml

```diff
@@ -12,19 +12,19 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v5
-      - name: Install nodejs
-        uses: actions/setup-node@v5
-        with:
-          node-version-file: .tool-versions
-          registry-url: "https://registry.npmjs.org"
 
       - name: Install pnpm
         uses: pnpm/action-setup@v4
+
+      - name: Install nodejs
+        uses: actions/setup-node@v6
         with:
-          version: latest
+          cache: pnpm
+          node-version-file: .tool-versions
```

Comment:
Should we update `.tool-versions` to `22`?

Reply:
We can switch to Node 24 straight away because it has been promoted to LTS recently. I suggest doing it in a separate PR because Node versioning is unrelated to pnpm. WDYT?

Reply:
Yes that will be nice!

Reply:
Sure! Feel free to merge this one if you're happy and I'll create a follow-up.